macOS/DuckDuckGo/DataImport/Logins/Chromium/ChromiumDataImporter.swift

```diff
@@ -105,7 +107,13 @@ internal class ChromiumDataImporter: DataImporter {
                                                    fraction: passwordsFraction + dataTypeFraction * 0.5))
 
             let bookmarksSummary = bookmarkResult.map { bookmarks in
-                bookmarkImporter.importBookmarks(bookmarks, source: .thirdPartyBrowser(source))
+                if featureFlagger.isFeatureOn(.updatedBookmarksFavoritesImport) {
+                    let newTabShortcuts = fetchShortcutsAsFavorites()
+                    let mergedBookmarks = FavoritesUtils.mergeBookmarksAndFavorites(bookmarks: bookmarks, favorites: newTabShortcuts)
+                    return bookmarkImporter.importBookmarks(mergedBookmarks, source: .thirdPartyBrowser(source), markRootBookmarksAsFavoritesByDefault: false)
+                } else {
+                    return bookmarkImporter.importBookmarks(bookmarks, source: .thirdPartyBrowser(source), markRootBookmarksAsFavoritesByDefault: true)
+                }
             }
 
             if case .success = bookmarksSummary {
```

Comment:
number of bookmarks reported by 
```
updateProgress(.importingBookmarks(numberOfBookmarks: try? bookmarkResult.get().numberOfBookmarks
```
wouldn't match actual number of imported bookmarks in case when we append new bookmarks from shortcuts in `mergeBookmarksAndFavorites`